src/main/resources/customImageTemplateWithManagedDisk.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "virtualNetworkName": "",
+    "virtualNetworkResourceGroupName": "",
+    "subnetName": "",
+    "nsgName": "",
+    "storageAccountName": "[concat('jnk',uniqueString(resourceGroup().id))]",
+    "vnetID": "[resourceId(variables('virtualNetworkResourceGroupName'), 'Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "publicIPAddressType": "Dynamic",
+    "storageAccountContainerName": "[concat('jnk',uniqueString(resourceGroup().id, deployment().name))]",
+    "storageAccountType": "Standard_LRS",
+    "jenkinsTag": "ManagedByAzureVMAgents",
+    "resourceTag": "AzureJenkinsMasterId",
+    "blobEndpointSuffix": ".blob.core.windows.net/"
```

Comment:
So the template cannot work in azure sovereign cloud like China or Germany?  I mean it will take us more time to debug and test if we need to support sovereign cloud.